documentation/feature/medical-system.md

```diff
@@ -6,238 +6,329 @@ group: feature
+
+
+- **Step 1:** Is the patient responsive?
+ - **Yes:** Ask him if he has wounds / he is in pain and act accordingly.
+ - **No:** Go to step 2.
+
+
+- <b>Step 2:</b> Does the patient have a pulse?
+ - **Yes:** Go to step 3.
+ - **No:** If you are alone provide CPR, if you have someone else get him to do CPR while you treat the patient's wounds. skip to step 3 or 4 depending on the situation.
  
-* Soldier:
-  * 4 x Bandage (basic)
-  * 3 x Bandage (elastic)
-  * 3 x Bandage (packing)
-  * 3 x QuikClot
-  * 1 x Morphine Autoinjector
-  * 1 x Tourniquet
-
-* Combat First Responder (CFR):
-  * 10-15 x Bandage (basic)
-  * 15-20 x Bandage (elastic)
-  * 5-8 x Bandage (packing)
-  * 10-15 x QuikClot
-  * 3 x Tourniquet
-  * 4 x Saline IV (500ml)
-  * 5 x Morphine Autoinjector
-  * 5 x Epinephrine Autoinjector
-  * 8 x Atropine Autoinjector
-
-* Medic:
-  * 10-15 x Bandage (basic)
-  * 15-20 x Bandage (elastic)
-  * 8 x Bandage (packing)
-  * 10-15 x QuikClot
-  * 5 x Tourniquet
-  * 6 x Saline IV (500ml)
-  * 8 x Morphine Autoinjector
-  * 8 x Epinephrine Autoinjector
-  * 12 x Atropine Autoinjector
-  * 1-3 x *Surgical kit*
-  * 1-3 x *Personal Aid Kit*
-
-* Paramedic:
-  * 10-15 x Bandage (basic)
-  * 15-20 x Bandage (elastic)
-  * 8 x Bandage (packing)
-  * 10-15 x QuikClot
-  * 5 x Tourniquet
-  * 2 x Saline IV (500ml)
-  * 3 x Blood IV (1000ml) - Taken from vehicle inventory only when needed
-  * 3 x Plasma IV (1000ml) - Taken from vehicle inventory only when needed
-  * 8 x Morphine Autoinjector
-  * 8 x Epinephrine Autoinjector
-  * 12 x Atropine Autoinjector
-  * 1-3 x *Surgical kit*
-  * 1-3 x *Personal Aid Kit*
-
-*medical item* - number of items that should be carried depends on module settings
-
-## Dependencies
+
+- **Step 3:** Is the patient wounded?
+ - **Yes**: Treat the wounds.
+ - **No:** Skip this step.
+  
+
+- **Step 4:** Did the patient lose blood?
+ - **Yes and normal blood pressure:** Use saline / blood IVs to restore the blood of the patient.
+ - **Yes and low blood pressure:** Use plasma IVs to stabilise the blood pressure and then use saline / blood IVs.
+ - **No:** Skip this step.
+
+
+- **Step 5:** Is the patient in pain?
+ - **Yes and stable pulse:** Give him morphine.
+ - **Yes and unstable heart rate:** Stabilize the heart rate before administrating morphine.
+ - **No:** You're done.
+
+
+- **Step 6:** is the patient awake now?
+ - **Yes:** You're done.
+ - **No:** Stabilize his pulse / make sure he isn't in pain or missing blood.
+
+Note that keeping the patient's vitals stable is always needed while treating him.
+
+#### 2.2.4 Additional informations
+
+- As an infantryman you can use a tourniquet to stop a limb from bleeding, note that this is supposed to be a temporary solution and leaving the tourniquet more than 5 minutes will induce pain cut and may kill you.
+- Epinephrine should **NEVER** be used in case of cardiac arrest.
```

Comment:
What are the consequences of giving epi while a player is in cardiac arrest? This documentation used to make quite a few claims like 'never give more than one morphine' that had no real connection to the code/actual system. 
Is this a left-over from that or is there actually some negative in-game effect of administering epi to somebody with cardiac arrest?